plugin-packs/postcss-preset-env/src/types/postcss-clamp/index.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module 'postcss-clamp' {
+	/** postcss-clamp plugin options */
+	export type pluginOptions = {
+		/** Preserve the original notation. default: false */
+		preserve?: boolean,
+		/**
+		 * The precalculate option determines whether values with the same unit should be precalculated.
+		 * default: false
+		 */
+		precalculate?: boolean,
+	};
+}
```

Comment:
Each untyped plugin has two files :

- `index.d.ts`
- `plugin-options.ts`

`index.d.ts` is used automatically by typescript when it can't find types from a package itself. This is only used internally before/during code compilation.

`plugin-options.ts` is how we can expose typed plugin options for untyped 3rd party plugins to our users.